common/rpc-directory/directory.mjs

```diff
@@ -293,8 +293,7 @@ class Directory {
 
     // TODO: Set them to a spawn point specified by map.json.
     // {MapCoord} mapCoord - Character's map coordinate.
-    setDefaultValue(playerData, 'mapCoord', new MapCoord('world1', 10, 10));
-
+    setDefaultValue(playerData, 'mapCoord', undefined);
```

Comment:
Why not initialize the mapCoord here? Why initialize it in `GatewayService`?

Reply:
We don't know which spawn point is empty, so we should use this._occupyCoord(firstLocation.mapCoord, playerID) to find it.